packages/guilded.js/lib/structures/Message.ts

```diff
@@ -105,6 +105,6 @@ export class Message extends Base<ChatMessagePayload> {
     /** Send a message that replies to this message. It mentions the user who sent this message. */
     reply(content: RESTPostChannelMessagesBody | string) {
         if (typeof content === "string") content = { content };
-        return this.client.messages.send(this.channelId, { ...content, replyMessageIds: content.replyMessageIds ?? [this.id] });
+        return this.client.messages.send(this.channelId, { ...content, replyMessageIds: [this.id] });
```

Comment:
Guilded allows replying to multiple messages, so we should support this.
```suggestion
        return this.client.messages.send(this.channelId, { ...content, replyMessageIds: content.replyMessageIds ? [this.id, ...content.replyMessageIds] : [this.id] });
```